_posts/en/newsletters/2020-08-19-newsletter.md

```diff
@@ -0,0 +1,164 @@
+---
+title: 'Bitcoin Optech Newsletter #111'
+permalink: /en/newsletters/2020/08/19/
+name: 2020-08-19-newsletter
+slug: 2020-08-19-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter describes a possible update to the BIP340
+specification of schnorr signatures and a new proposed BIP that
+specifies how Bitcoin nodes should handle P2P protocol feature
+negotiation in a forwards-compatible way.  Also included are our
+regular sections with notable changes to services and client software,
+releases and release candidates, and changes to popular Bitcoin
+infrastructure software.
+
+## Action items
+
+*None this week.*
+
+## News
+
+- **Proposed uniform tiebreaker in schnorr signatures:** the [elliptic
+  curve cryptography][] used in Bitcoin requires identifying points on
+  Elliptic Curves (ECs).  This can be done unambiguously using a 32-byte X
+  coordinate and a 32-byte Y coordinate.  However, if you know a point's
+  X coordinate, then you can calculate the only two possible locations
+  for its Y coordinate.  This small ambiguity can be resolved either by
+  providing a single bit of extra data to indicate which of the two
+  coordinates to use or by using a type of method known as a
+  *tiebreaker* to automatically choose one Y coordinate or the other for
+  each X coordinate.
+
+    The [BIP340][] specification of schnorr signatures for Bitcoin
+    currently uses a tiebreaker method to minimize the amount of data
+    needed to identify EC points in [taproot][topic taproot] public keys
```

Comment:
I think just "public keys and signatures" is better than "taproot public keys and schnorr signatures" here.

Reply:
Why?  (I'm curious, and because my thinking was to make it clear this doesn't change anything for existing SEC-style pubkeys and DER-encoded signatures).

Reply:
Because BIP340 is separate from taproot, so these changes really apply to all BIP340 pubkeys and signatures. I don't think the word taproot is needed to qualify pubkeys, and since BIP 340 is all about schnorr signatures, I don't think the word schnorr is needed to qualify signatures.